maps/aurora/code/aurora_shuttles.dm

```diff
@@ -155,4 +155,16 @@
 	shuttle.area_offsite = locate(/area/shuttle/legion/centcom)
 	shuttle.area_station = locate(/area/shuttle/legion/station)
 	shuttles["Tau Ceti Foreign Legion"] = shuttle
+	START_PROCESSING(shuttle_controller, shuttle)
+
+	//Away Site shuttle.
+
+	shuttle = new()
+	shuttle.location = 0
+	shuttle.warmup_time = 10
+	shuttle.area_station = locate(/area/shuttle/research/station)
+	shuttle.area_offsite = locate(/area/shuttle/research/outpost)
+	shuttle.docking_controller_tag = "science_shuttle"
+	shuttle.dock_target_station = "science_bridge"
+	shuttles["Research Away Shuttle"] = shuttle
```

Comment:
That should be changed to just `Research` so we can reuse the research shuttle console.